src/constants/constants.js

```diff
@@ -1,4 +1,7 @@
 export const PAGE_RANGE = 2;
-export const ITEMS_PER_PAGE = 12;
+export const ITEMS_PER_PAGE = 15;
 export const INITIAL_PAGE = 1;
+export const PRODUCTS_URL = '/res/products.json';
+export const CAROUSEL_PRODUCTS_URL = '/res/carouselProducts.json';
+
```

Comment:
check if it can be optimized

Reply:
Will check and try to implement in the upcoming PRs till then requesting you to mark this resolved.